package.json

```diff
@@ -15,13 +15,22 @@
     "uuid": "3.2.1"
   },
   "devDependencies": {
+    "editorconfig": "^0.15.3",
+    "eslint": "5.16.0",
+    "eslint-plugin-jsdoc": "3.8.0",
+    "eslint-plugin-mocha": "5.2.0",
+    "eslint-plugin-security": "1.4.0",
     "expect.js": "0.3.1",
+    "istanbul": "^0.4.5",
```

Comment:
@Dhroov7 Why do we need istanbul and nyc?

Reply:
We can remove istanbul but it gives code coverage report.
I've removed nyc.